src/state/newActions.js

```diff
@@ -0,0 +1,62 @@
+import { createAction } from 'redux-actions'
+import { createPayload } from 'utils/reduxUtils'
+
+export const addList = createAction(
+  'ADD_LIST',
+  createPayload('list'),
+)
+
+export const updateList = createAction(
+  'UPDATE_LIST',
+  createPayload('listId', 'change'),
+)
+
+export const deleteList = createAction(
+  'DELETE_LIST',
+  createPayload('listId'),
+)
+
+export const addListItem = createAction(
+  'ADD_LIST_ITEM',
+  createPayload('listId', 'item'),
+)
+
+export const updateListItem = createAction(
+  'UPDATE_LIST_ITEM',
+  createPayload('listId', 'itemId', 'change'),
+)
+
+export const deleteListItem = createAction(
+  'DELETE_LIST_ITEM',
+  createPayload('listId', 'itemId'),
+)
+
+export const addListItemNote = createAction(
+  'ADD_LIST_NOTE',
+  createPayload('listId', 'itemId', 'note'),
+)
+
+export const updateListItemNote = createAction(
+  'UPDATE_LIST_NOTE',
+  createPayload('listId', 'itemId', 'noteId', 'change'),
+)
+
+export const deleteListItemNote = createAction(
+  'DELETE_LIST_NOTE',
+  createPayload('listId', 'itemId', 'noteId'),
+)
+
+export const setActiveList = createAction(
+  'SET_ACTIVE_LIST',
+  createPayload('listId'),
+)
+
+export const setFavouriteList = createAction(
+  'SET_FAVOURITE_LIST',
+  createPayload('listId'),
+)
+
+export const setIsModalOpen = createAction(
+  'SET_IS_MODAL_OPEN',
+  createPayload('isOpen'),
+)
```

Comment:
Can you create an object of the constants in this file that you export instead of individual exports?